bin/compile

```diff
@@ -143,7 +144,31 @@ export CPPPATH="$INCLUDE_PATH"
 export PKG_CONFIG_PATH="$BUILD_DIR/.apt/usr/lib/x86_64-linux-gnu/pkgconfig:$BUILD_DIR/.apt/usr/lib/i386-linux-gnu/pkgconfig:$BUILD_DIR/.apt/usr/lib/pkgconfig:$PKG_CONFIG_PATH"
 
 #give environment to later buildpacks
-export | grep -E -e ' (PATH|LD_LIBRARY_PATH|LIBRARY_PATH|INCLUDE_PATH|CPATH|CPPPATH|PKG_CONFIG_PATH)='  > "$LP_DIR/export"
+topic "Exporting environment"
+BUILDPACK_EXPORT_FILE="$BP_DIR/export"
+export | grep -E -e ' (PATH|LD_LIBRARY_PATH|LIBRARY_PATH|INCLUDE_PATH|CPATH|CPPPATH|PKG_CONFIG_PATH)=' > "$BUILDPACK_EXPORT_FILE"
 
 topic "Rewrite package-config files"
 find "$BUILD_DIR/.apt" -type f -ipath '*/pkgconfig/*.pc' -print0 | xargs -0 --no-run-if-empty -n 1 sed -i -e 's!^prefix=\(.*\)$!prefix='"$BUILD_DIR"'/.apt\1!g'
+
+# post-installation logic for git
+# check is git is in the Aptfile and set GIT_EXEC_PATH
+if grep -qw "git" "${BUILD_DIR}/Aptfile"; then
+  if command -v git >/dev/null 2>&1; then
+    GIT_CORE_DIR="${BUILD_DIR}/.apt/usr/lib/git-core"
+    if [[ -d "$GIT_CORE_DIR" ]]; then
+      topic "Setting GIT_EXEC_PATH to $GIT_CORE_DIR"
+      # set env var in runtime profile script
+      # shellcheck disable=SC2016
+      echo 'export GIT_EXEC_PATH="$HOME/.apt/usr/lib/git-core"' >> "$RUNTIME_PROFILE_SCRIPT"
+      # export the env var within the build
+      export GIT_EXEC_PATH=$GIT_CORE_DIR
+      # export the env var to later buildpacks
+      export | grep -E -e ' (GIT_EXEC_PATH)=' >> "$BUILDPACK_EXPORT_FILE"
+    else
+      topic "Git core directory $GIT_CORE_DIR is either missing or empty. Skipping GIT_EXEC_PATH setup."
+    fi
+  else
+    error "Git installation failed. Check Aptfile and logs."
```

Comment:
Do you know under what conditions this would occur? It seems apt would have exited non-zero if the install had failed, so this seems redundant?

Reply:
Fixed in https://github.com/heroku/heroku-buildpack-apt/pull/143/commits/e6f7e4f89f4e847b0ac2d8440ed69716d484b96e